### ISSUE TITLE
[Serious] Animated Title Bug 

### ISSUE THREAD

**Issue by MrTeeXD** [OpenShot/openshot-qt#2330]
**Describe the bug**
When an animated title is rendered in Openshot via the Blender API, there is a bug that occurs when its placed on the timeline

**System Details (please complete the following information):**
 - Operating System / Distro: [Windows 10]
 - OpenShot Version [2.4.3 Latest Daily Build]

**To Reproduce**
1. Go to "Title"
2. Click on "Animated Title"
3. Scroll down and select your prefered Title to render
4. Place your tile on the timeline
5: A bug will happen when the whole thing is played

**Expected behavior**
The Animated title will play smoothly with no lag 

**Screenshots**
[2018-11-06 19-20-59.zip](https://github.com/OpenShot/openshot-qt/files/2554298/2018-11-06.19-20-59.zip)

**Logs**
If you are experiencing a crash, please collect and attach logs of the problem.

**Additional context**
I believe this is in all the daily builds. Please see the attached Recording of this bug.

**Comment by DylanC** [comment]
@MisterTeeXD - Thanks, I will have a look soon.

**Comment by DylanC** [comment]
@MisterTeeXD - I was not able to play your recording for some reason. Do you have different format than .flv?

I did try this myself with the daily build and I didn't come across a problem. (yet)

**Comment by MisterTeeXD** [comment]
Can you try to Convert it to .flv, i am not at my PC

**Comment by DylanC** [comment]
@MisterTeeXD - Converted to mp4 and I can view it now. Indeed it does look slow. What spec is your PC?

**Comment by MisterTeeXD** [comment]
Windows 10, 64bit, Intel i5 Processor, 4GB RAM and Integrated GPU

**Comment by ferdnyc** [comment]
Two questions:

1. Does this _actually_ happen with all animated titles **in all cases**, or was it specific to the project you were working on at the time?

    Like, if you just launch OpenShot, with nothing loaded in Project Files or on the Timeline, then go **directly** to Title > Animated Title in the menus, render one of the choices, grow a 4" long beard waiting for it to finish, drag the rendered titles to the timeline (or use the "Add to Timeline..." context menu option), and hit Play in the preview window... is _that_ still just as slow?

    I just did precisely that, in 2.4.3 on my Core i5-2400 / 8GB / (cheapo) Nvidia GPU Linux system, using the Neon Curves template with all-default values (right down to "Title" as the title text). It was a _little_ laggy the first time through as it generated the cache (but only a little), then completely fine after that.

2. If the issue _is_ specific to the project you were working on at the time, is that project saved to disk? What's the file size of the `.osp` file?

**Comment by MisterTeeXD** [comment]
1: Yes it does happen with all of the animated titles
2: I think its 1MB

**Comment by ferdnyc** [comment]
Hmm. 1MB is quite excessive for an `.osp` file. The project file I just saved, containing nothing but a single Animated Title render, is 25 KB. (The project file only contains structure, all of the media files are stored separate from the `.osp` file itself, even titles.)

Still, I just downloaded the latest daily build and tried again, this time with the Dissolving Text template, thinking it would render faster since it's only 128 frames. (Joke was on me, that particular title renders _sooooo_ slowly that 128 frames took WAY longer than the 250 frames of Neon Curves.)

This time around, OpenShot is screwing up the display of every frame past the first (I think because they're transparent, and it can't auto-detect the size properly)... but it _previews_ incredibly quickly.

So, unfortunately this issue doesn't appear to be reproducible, at least on Linux. I don't have access to a Win 10 machine to test there, I'm afraid.
